codazen/webDiplomacy
# Fix transition image

To test: 

Pull this branch, refresh page or let phase transition, verify loading image is shown properly.
